specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2024-03-03/gallery.json

```diff
@@ -2435,6 +2435,10 @@
         },
         "securityProfile": {
           "$ref": "#/definitions/ImageVersionSecurityProfile"
+        },
+        "restore": {
+          "type": "boolean",
```

Comment:
Could we update at least 1 example to utilize this new property

Reply:
added